src/openrct2/interface/StdInOutConsole.cpp

```diff
@@ -17,6 +17,9 @@
 
 #include <linenoise.hpp>
 
+// Ignore isatty warning on WIN32
+#pragma warning(disable : 4996)
```

Comment:
Should this perhaps be wrapped with macro guards? While some compilers understand `#pragma` the warning is probably msvc specific.

Reply:
Good idea. What about the following?
```suggestion
#ifdef _MSC_VER
#pragma warning(disable : 4996)
#endif
```